### DISCUSSION TITLE
JS Methods list

### DISCUSSION THREAD

**Discussion by 350d** [obs-websocket-community-projects/obs-websocket-js#269] [Q&A]
Hello! I've developed my project year or so ago and now I can't find the page where all available JS methods was listed.
Can you please point me to this page? Thank you!

**Comment by t2t2** [comment]
For obs-websocket-js API you'd want to check the tag which matches the version you're using - https://github.com/obs-websocket-community-projects/obs-websocket-js/tags

For methods/events that can be called from obs-websocket check the tag matching release of obs-websocket, eg for v4.9.1 (current stable) https://github.com/obsproject/obs-websocket/blob/4.9.1/docs/generated/protocol.md
